### ISSUE TITLE
Updating make-up of technical committee

### ISSUE THREAD

**Issue by paulromano** [openmc-dev/openmc#3217]
I'd like to propose some changes to the @openmc-dev/technical-committee, which has stayed the same since it was created. First I have talked to @smharper, who has agreed to step down from the technical committee given that he has not been actively involved in development. In his please, I would like to nominate @pshriwise to the technical committee, who has been contributing to OpenMC since 2018 and is the [number 2 contributor](https://github.com/openmc-dev/openmc/graphs/contributors) at this point. He has been responsible for many features, bug fixes, and enhancements that cover a wide range of the codebase.

@smharper @nelsonag @bforget Do you agree with the above changes to the technical committee?

**Comment by smharper** [comment]
I agree with the changes, especially given the long time since I have contributed. Congratulations on the recognition, @pshriwise !

**Comment by bforget** [comment]
I agree as well!
________________________________

**Comment by nelsonag** [comment]
I agree and congratulations to @pshriwise !

**Comment by tomfieno** [comment]
Done

[image: image.png]

**Comment by paulromano** [comment]
I was just reminded of this issue and do want to propose a few more changes. First, I'll start by nominating @shimwell to the technical committee, who has been a long-time contributor (#&NoBreak;5 overall and #&NoBreak;1 European 🇪🇺 ) _and_ our chief evangelist 😄 

@pshriwise @nelsonag @bforget do you agree with adding @shimwell to our technical committee?

**Comment by nelsonag** [comment]
I definitely agree!

**Comment by bforget** [comment]
I agree!

**Comment by pshriwise** [comment]
Agreed!

**Comment by paulromano** [comment]
My next, and I think final, proposal for a change is to replace @bforget on the technical committee with @jtramm. Ben has been an incredible supporter of OpenMC, first as my PhD advisor when he was crazy enough to allow me to start a new Monte Carlo code and then continuing over the years with many R&D contributions through his students (windowed multipole, depletion, parallelism, random ray, etc.). However, he hasn't been directly involved in OpenMC code development for some time and the expectation for TC members is that they should be actively reviewing, submitting PRs, and participating in discussions around code contributions.

@jtramm is the #​7 overall contributor and #​5 contributor in the last 2 years, but I would say that undersells the impact of his contributions in OpenMC. First, his work on GPU porting through the ExaSMR project showed definitely that MC on GPUs for a production code was not a pipe dream but something achievable. Through a newly funded ARPA-E project, he will be working on bringing those capabilities into our main branch. John also invented and implemented the random ray method in OpenMC along with and associated variance reduction methods. Together, his contributions have shaped the direction of our research and garnered a lot of attention for the code, keeping it at the forefront of Monte Carlo code development.

@pshriwise @nelsonag @bforget @shimwell do you agree with the proposed change to the TC?

**Comment by pshriwise** [comment]
Agreed!

**Comment by bforget** [comment]
I fully support this as well!

**Comment by shimwell** [comment]
Agreed

**Comment by nelsonag** [comment]
I agree with all!
